base_tier_validation/__manifest__.py

```diff
@@ -1,14 +1,16 @@
 # Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "12.0.3.1.0",
+    "version": "12.0.4.1.0",
```

Comment:
You don't need to bump version. As new rule, the person who merge will use ocabot to bump it.